ecosystem/node-checker/src/server/run.rs

```diff
@@ -84,31 +80,36 @@ pub async fn run(args: Run) -> Result<()> {
         allow_preconfigured_test_node_only: args.allow_preconfigured_test_node_only,
     };
 
-    let api_service =
-        build_openapi_service(api, args.listen_address.clone(), args.listen_port, None);
+    let api_endpoint = format!("/{}", args.server_args.api_endpoint);
```

Comment:
Is there anything specifying that api_endpoint doesn't start with a `/`?  Should we just  enforce it in the input?